test/e2e/scenarios/node-labels.go

```diff
@@ -79,15 +79,26 @@ func labeledDeployTestSuite() {
 			}
 		}
 
+		// wait till operator reconcile csi
+		// operator has to receive NodeUpdate request and label new nodes to expand node-daemonset
+		// if new pods aren't created in time, WaitForPodsRunningReady will be skipped
+		deadline := time.Now().Add(time.Second * 30)
+		for {
+			np, err = common.GetNodePodsNames(f)
+			if err != nil {
+				ginkgo.Fail(err.Error())
+			}
+			if len(np) == len(nodes) {
+				break
+			}
+			if time.Now().After(deadline) {
+				ginkgo.Fail(fmt.Sprintf("Pods number is %d, should be %d", len(np), len(nodes)))
+			}
+			time.Sleep(time.Second * 3)
+		}
+
 		err = e2epod.WaitForPodsRunningReady(f.ClientSet, f.Namespace.Name, 0, 0,
```

Comment:
Can we pass min pods count to `WaitForPodsRunningReady` instead? from method description:
```
// WaitForPodsRunningReady waits up to timeout to ensure that all pods in
// namespace ns are either running and ready, or failed but controlled by a
// controller. Also, it ensures that at least minPods are running and
// ready.
```

Reply:
Ok, got it.